modules/apps/kate/default.nix

```diff
@@ -196,25 +196,65 @@ in
     '';
   };
 
+  # ==================================
+  #     BRACKETS
+  options.programs.kate.editor.brackets = {
+    characters = lib.mkOption {
+      type = lib.types.str;
+      default = "<>(){}[]'\"\`";
+      example = "<>(){}[]'\"\`*_~";
+      description = "Which characters kate will treat as brackets.";
+    };
+    automaticallyAddClosing = lib.mkEnableOption {
+      description = "that a closing bracket is automatically inserted upon typing the opening";
+    };
+    highlightRangeBetween = lib.mkEnableOption {
+      description = ''
+        that the range between the brackets is highlighted when the cursor is adjacent
```

Comment:
Same here

Reply:
Oh. I misunderstood what the default description of `mkEnableOption` does. Will fix.

Reply:
I was mainly talking about the punctuation and capitalization in the sentence, but you also don't need the description field I don't think. For example `lib.mkEnableOption ''That the range between the brackets is highlighted when the cursor is adjacent.''` would be better (though maybe written slightly differently as the formulation seems a little weird to me).

Reply:
yes, i thought it would automatically put "whether to enable ${my part}." into the description. I fixed it now.